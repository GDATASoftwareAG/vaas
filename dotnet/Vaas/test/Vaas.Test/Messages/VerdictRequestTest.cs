using System.Text.Json;
using Snapshooter.Xunit;
using Vaas.Messages;
using Xunit;

namespace Vaas.Test.Messages;

public class VerdictRequestTest
{
    [Fact]
    public void Serialize()
    {
        var json = JsonSerializer.Serialize(new VerdictRequest("", ""));
        Snapshot.Match(json);
    }
    
    [Fact]
    public void Serialize_WithOptions()
    {
        var json = JsonSerializer.Serialize(new VerdictRequest("", "") { UseCache = false, UseShed = false });
        Snapshot.Match(json);
    }
}