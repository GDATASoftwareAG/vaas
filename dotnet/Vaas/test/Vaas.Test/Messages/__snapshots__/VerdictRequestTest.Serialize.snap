{"kind":"VerdictRequest","sha256":"","guid":"5208034c-44f2-4b8e-a96b-4c05bd663224","session_id":"","verdict_request_attributes":null}
